extensions/cql/standard/schema/examples/example02.txt

```diff
@@ -1,3 +1,3 @@
-EXAMPLE 2: Any item where eo:instrument starts with OLI 
+EXAMPLE: Find any item where the instrument name starts with 'OLI'.
 
 filter=eo:instrument like 'OLI*'
```

Comment:
Shouldn't this be something like `filter=eo:instrument like '^OLI'`, if we use regex?